xml/System.Security.Cryptography/SafeEvpPKeyHandle.xml

```diff
@@ -260,11 +260,21 @@
         <Parameter Name="keyUri" Type="System.String" Index="1" FrameworkAlternate="net-9.0" />
       </Parameters>
       <Docs>
-        <param name="providerName">To be added.</param>
-        <param name="keyUri">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="providerName">The name of the <code>OSSL_PROVIDER</code> to process the key open request.</param>
+        <param name="keyUri">The URI assigned by the <code>OSSL_PROVIDER</code> of the key to open.</param>
+        <summary>Open a named key using a named <code>OSSL_PROVIDER</code>.</summary>
+        <returns>The opened key.</returns>
+        <remarks>
+          <para>Both <paramref name="providerName" /> and <paramref name="keyUri" /> must be trusted inputs.</para>
+          <para>This operation will fail if OpenSSL cannot successfully load the named <code>OSSL_PROVIDER</code>, or if the named <code>OSSL_PROVIDER</code> cannot load the named key.</para>
+          <para>The syntax for <paramref name="keyUri" /> is determined by each individual named <code>OSSL_PROVIDER</code>.</para>
+        </remarks>
+        <exception cref="T:System.ArgumentNullException">
+          <paramref name="providerName" /> or <paramref name="keyUri" /> is <see langword="null" />.</exception>
+        <exception cref="T:System.ArgumentException">
+          <paramref name="providerName" /> or <paramref name="keyUri" /> is the empty string.</exception>
+        <exception cref="T:System.PlatformNotSupportedException">The current platform does not support OpenSSL Providers.</exception>
```

Comment:
This exception was not documented, but was introduced in https://github.com/dotnet/runtime/pull/106397. I documented it here.